server/lib/utils/connstr-mysql.js

```diff
@@ -0,0 +1,33 @@
+import URI from "urijs";
+
+export function parseConnectionConfig(settings) {
```

Comment:
This is related to `mysql` and `postgres`/`redshift`, so I externalized it rather than writing the same code twice. I will go and rename it though to make it clearer.